### PR TITLE
Component: Forum post PR

### PR DESCRIPTION
**Pull request by torykling**
I built the forum post from issue #80. 

Questions:
Should this be a functional component? I was thinking it might need to fetch data in a component did mount, but then thought a parent component would do that.  

This takes a ton of props, which feels messy. Is that fine?

### CODE REVIEW

File: client/src/components/ForumPost/ForumPost.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+.ForumPost__top-middle {
+  border: 1px solid rgba(37, 37, 37, 0.5);
+  border-radius: 3px;
+  padding-bottom: 0.5rem;
+}
+.ForumPost .Button {
+  border: 1px solid rgba(37, 37, 37, 0.5);
+  border-radius: 3px;
+  font-size: 1rem;
+}
```
Comment (erinkelley27) [inline client/src/components/ForumPost/ForumPost.css]: Combine the repeated styles into one declaration block with multiple selectors.

File: client/src/components/ForumPost/ForumPost.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  background-color: var(--gray-lite);
+}
+.ForumPost__topic-pills {
+  display: flex;
+  justify-content: space-evenly;
+}
+.ForumPost__likes-comments {
+  display: flex;
+  justify-content: space-evenly;
+}
```
Comment (erinkelley27) [inline client/src/components/ForumPost/ForumPost.css]: Combine into one declaration block with multiple selectors.

File: client/src/components/ForumPost/ForumPost.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+import PostBody from "../PostBody/PostBody.js";
+import Button from "../Button/Button.js";
+import ProfilePicture from "../ProfilePicture/ProfilePicture.js";
+import Image from "../../../../assets/images/profile_large.png";
+import heart from "../../../../assets/images/heart_black.png";
+import comment from "../../../../assets/images/comment_black.png";
+import Pill from "../Pill/Pill.js";
+import PropTypes from "prop-types";
+
+class ForumPost extends Component {
```
Comment (erinkelley27) [inline client/src/components/ForumPost/ForumPost.js]: Make this a functional component.

Comment (torykling) [reply client/src/components/ForumPost/ForumPost.js]: Sounds good. Thanks Erin!

**Pull request merged**